ui/app/services/store.js

```diff
@@ -138,12 +138,22 @@ export default class StoreService extends Store {
     return resp;
   }
 
+  forceUnload(modelName) {
+    // Hack to get unloadAll to work correctly until we update to ember-data@4.12
+    // so that all the records are properly unloaded and we don't get ghost records
+    this.peekAll(modelName).length;
+    // force destroy queue to flush https://github.com/emberjs/data/issues/5447
+    run(() => this.unloadAll(modelName));
+  }
+
   // pushes records into the store and returns the result
   fetchPage(modelName, query) {
     const response = this.constructResponse(modelName, query);
-    this.unloadAll(modelName);
+    this.forceUnload(modelName);
+    // Hack to ensure the pushed records below all get in the store
+    this.peekAll(modelName).length;
```

Comment:
I'm guessing this is necessary but since there is already a peek in the `forceUnload` method it has to be done again after `unloadAll`?